test/integration/controller_test.go

```diff
@@ -1025,6 +1026,26 @@ func prependGetSecretReaction(fakeKubeClient *fake.Clientset, secretName string,
 	})
 }
 
+func prependGetNamespaceReaction(fakeKubeClient *fake.Clientset, namespace string) {
+	fakeKubeClient.PrependReactor("get", "namespaces", func(action clientgotesting.Action) (bool, runtime.Object, error) {
+		getAction, ok := action.(clientgotesting.GetAction)
+		if !ok {
+			return true, nil, apierrors.NewInternalError(fmt.Errorf("could not convert get namespace action to a GetAction: %T", action))
+		}
+		if getAction.GetName() != namespace {
+			return false, nil, nil
+		}
+		ns := &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      namespace,
+				Namespace: namespace,
+				UID:       "testnamespace1234",
```

Comment:
you need to do that also here:
https://github.com/kubernetes-sigs/service-catalog/blob/master/pkg/controller/case_test.go#L87-L91

sth like:
```go

import "k8s.io/apimachinery/pkg/util/uuid"

k8sClient.CoreV1().Namespaces().Create(&corev1.Namespace{
		ObjectMeta: metav1.ObjectMeta{
			Name: testNamespace,
			UID:  uuid.NewUUID(), // fake clientset doesn't  duplicate server-side behavior like uid assignment etc
		},
	})
```